backend/app/libs/coingecko/coins.py

```diff
@@ -34,21 +33,12 @@ async def _get_data(
         # replace ETH address to WETH
         if token_address == "0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee":
             token_address = "0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2"
-        try:
-            resp = await client.get(
-                COINGECKO_API_URL
-                + f"/coins/ethereum/contract/{token_address}/market_chart/range"
-                + f"?vs_currency=usd&from={start_date.timestamp()}&to={end_date.timestamp()}",
-                timeout=timeout,
-            )
-        except MaybeEncodingError:
-            sleep(5)
-            resp = await client.get(
-                COINGECKO_API_URL
-                + f"/coins/ethereum/contract/{token_address}/market_chart/range"
-                + f"?vs_currency=usd&from={start_date.timestamp()}&to={end_date.timestamp()}",
-                timeout=timeout,
-            )
+        resp = await client.get(
+            COINGECKO_API_URL
+            + f"/coins/ethereum/contract/{token_address}/market_chart/range"
+            + f"?vs_currency=usd&from={start_date.timestamp()}&to={end_date.timestamp()}",
+            timeout=timeout,
+        )
         sleep(5)
```

Comment:
Can't we remove this? Can be done in another PR.

Reply:
Can you elaborate a bit more on what you'd like to remove?

Reply:
I meant line 42 with the `sleep(5)`